numba_cuda/numba/cuda/api.py

```diff
@@ -95,6 +97,17 @@ def is_float16_supported():
     return True
 
 
+def is_bfloat16_supported():
+    """Whether bfloat16 are supported.
+
+    bfloat16 are only supported on devices with compute capability >= 8.0 and cuda version >= 12.0
+    """
+    return current_context().device.supports_bfloat16 and cuda_version >= (
```

Comment:
Instead, we should get cuda version from here:
```suggestion
    """
    cuda_version = get_version()
    return current_context().device.supports_bfloat16 and cuda_version >= (
```